frontend/Store.js

```diff
@@ -20,6 +20,9 @@ var serializePropsForCopy = require('../utils/serializePropsForCopy');
 var invariant = require('./invariant');
 var SearchUtils = require('./SearchUtils');
 var ThemeStore = require('./Themes/Store');
+const {get, set} = require('../utils/storage').session;
+
+const SESSION_STORAGE_TRACE_UPDATES_ENABLED_KEY = 'traceUpdates:enabled';
```

Comment:
nit: other keys are just named after the preference, e.g. `themeName` or the location+preference `profiler:commitThreshold`. I think ":enabled" is implied. 😄

Reply:
@bvaughn OK, thanks, will change that